src/forms/FormLines.tsx

```diff
@@ -0,0 +1,44 @@
+import { Css } from "src/Css";
+
+export interface FormLinesProps {
+  /** Let the user interleave group-less lines and grouped lines. */
+  children: JSX.Element[];
+  width?: "md" | "sm";
+}
+
+/**
+ * Applies standard Form layout/size/spacing between lines.
+ *
+ * Lines can either be individual form fields, or a group of form fields
+ * (see the `FieldGroup` component), where they will be laid out side-by-side.
+ */
+export function FormLines(props: FormLinesProps) {
+  const { children, width = "md" } = props;
+  return (
+    <div
+      css={{
+        ...Css.df.flexColumn.wPx(sizes[width]).$,
+        // Purposefully use this instead of childGap3 to put margin-bottom on the last line
+        "& > *": Css.mb3.$,
+      }}
+    >
+      {children}
+    </div>
+  );
+}
+
+/** Groups multiple fields side-by-side. */
+export function FieldGroup(props: {
```

Comment:
**Question:** Is this like FormColumn?

Reply:
Well, it's more like "a multi-field line", i.e.:

```
| first name          | // 1st line
| last name          | // 2nd line
| middle | phone  | // a FieldGroup one the 3rd line
```